com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/EmissionUIBlock.cs

```diff
@@ -108,33 +103,37 @@ public override void LoadMaterialProperties()
             UVMappingMaskEmissive = FindProperty(kUVMappingMaskEmissive);
         }
 
-        internal static void UpdateEmissiveColorFromIntensityAndEmissiveColorLDR(MaterialEditor materialEditor, Material[] materials)
+        internal static void UpdateEmissiveColorLDRAndIntensityFromEmissiveColor(MaterialProperty emissiveColorLDR, MaterialProperty emissiveIntensity, MaterialProperty emissiveColor)
         {
-            materialEditor.serializedObject.ApplyModifiedProperties();
-            foreach (Material target in materials)
-            {
-                target.UpdateEmissiveColorFromIntensityAndEmissiveColorLDR();
-            }
-            materialEditor.serializedObject.Update();
-        }
+            // specifies the max byte value to use when decomposing a float color into bytes with exposure
+            // this is the value used by Photoshop
+            const byte k_MaxByteForOverexposedColor = 191;
```

Comment:
what is this code doing? Same remark than above, should we have a backport for it?

Reply:
yes, it converts emissive hdr to ldr + intensity when ticking "use emissive intensity"